### ISSUE TITLE
support MVAR table

### ISSUE THREAD

**Issue by robmck-ms** [fonttools/fonttools#689]
It would be helpful if fonttools supported the MVAR table from the OpenType 1.8 spec:
https://www.microsoft.com/typography/otspec/MVAR.htm

**Comment by anthrotype** [comment]
Behdad pushed b4d53811e904bd8315639b364a9d9747bb08350c yesterday.
It would be nice to have some unittest before tagging a release with MVAR support.

**Comment by behdad** [comment]
Here's the MVAR from a Selawik test Rob sent me before (I had to fix the offset length), in case someone wants to make it into a test:

```
  <MVAR>
    <Version value="0x00010000"/>
    <!-- AxisCount=1 -->
    <ValueRecordSize value="8"/>
    <!-- ValueRecordCount=7 -->
    <VarStore Format="1">
      <Format value="1"/>
      <VarRegionList>
        <!-- RegionAxisCount=1 -->
        <!-- RegionCount=1 -->
        <Region index="0">
          <VarRegionAxis index="0">
            <StartCoord value="0.0"/>
            <PeakCoord value="1.0"/>
            <EndCoord value="1.0"/>
          </VarRegionAxis>
        </Region>
      </VarRegionList>
      <!-- VarDataCount=1 -->
      <VarData index="0">
        <!-- ItemCount=4 -->
        <NumShorts value="1"/>
        <!-- VarRegionCount=1 -->
        <VarRegionIndex index="0" value="0"/>
        <Item index="0" value="[-200]"/>
        <Item index="1" value="[-50]"/>
        <Item index="2" value="[100]"/>
        <Item index="3" value="[200]"/>
      </VarData>
    </VarStore>
    <ValueRecord index="0">
      <ValueTag value="hasc"/>
      <VarIdx value="3"/>
    </ValueRecord>
    <ValueRecord index="1">
      <ValueTag value="hcla"/>
      <VarIdx value="3"/>
    </ValueRecord>
    <ValueRecord index="2">
      <ValueTag value="hcld"/>
      <VarIdx value="3"/>
    </ValueRecord>
    <ValueRecord index="3">
      <ValueTag value="hdsc"/>
      <VarIdx value="0"/>
    </ValueRecord>
    <ValueRecord index="4">
      <ValueTag value="hlgp"/>
      <VarIdx value="2"/>
    </ValueRecord>
    <ValueRecord index="5">
      <ValueTag value="sbyo"/>
      <VarIdx value="1"/>
    </ValueRecord>
    <ValueRecord index="6">
      <ValueTag value="spyo"/>
      <VarIdx value="2"/>
    </ValueRecord>
  </MVAR>
```

**Comment by moyogo** [comment]
@behdad I made a test from that XML, but I came up with an issue with AxisCount being 0 instead of 1 when compiling. AxisCount is not used later, RegionAxisCount is used instead and has the correct value.

Both [AxisCount](https://www.microsoft.com/typography/otspec/MVAR.htm) and [RegionAxisCount](https://www.microsoft.com/typography/otspec/otvarcommonformats.htm) are defined as "The number of variation axes for this font. This must be the same number as axisCount in the 'fvar' table."
